index.js

```diff
@@ -154,6 +154,16 @@ module.exports = function (host, port) {
       return send('add', null, null, file, cb)
     },
     cat: argCommand('cat'),
+    slurp: function (path, cb) {
+      this.cat(path, function (err, stream) {
+        if (err) return cb(err)
+        var buf = ''
+        stream
+          .on('error', cb)
+          .on('data', function (data) { buf += data })
+          .on('end', function () { cb(null, buf) })
+      })
+    },
```

Comment:
I wouldn't put this on the "main" scope of the API like @jbenet said. It's not really part of IPFS and is more a helper function. Also, it does not respect buffers as it's going to call `toString` due to the plus operator concatenation. I don't know if this is really an issue due to the way javascript works but something we should at least look into.